### PR TITLE
Improve language and formatting of OCA Guidelines

### PR DESCRIPTION
**Pull request by rz-comp**
- Improve wording and grammar
- Improve some markdown formatting
- Limit line length to 80 characters where possible

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Wow, thanks for the huge redaction review you have made.

:+1:

**Comment by rvalyi** [comment]
:+1:  thanks

**Comment by max3903** [comment]
@rz-comp Thanks! Very much appreciated.
:+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -590,18 +604,17 @@ changes. This part should be multiple lines no longer than 80 characters.
 * Merge proposals should follow the same rules as the title of the propsal is
```
Comment (elicoidal) [inline CONTRIBUTING.md]: s/propsal/proposal

**Comment by elicoidal** [comment]
A must! thanks!

**Pull request merged**